dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/ITextToImageService.cs

```diff
@@ -16,15 +20,15 @@ public interface ITextToImageService : IAIService
     /// <summary>
     /// Generate an image matching the given description
     /// </summary>
-    /// <param name="description">Image description</param>
+    /// <param name="prompt">Image generation prompt</param>
     /// <param name="width">Image width in pixels</param>
     /// <param name="height">Image height in pixels</param>
     /// <param name="kernel">The <see cref="Kernel"/> containing services, plugins, and other state for use throughout the operation.</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>Generated image in base64 format or image URL</returns>
     [Experimental("SKEXP0001")]
     public Task<string> GenerateImageAsync(
-        string description,
+        string prompt,
```

Comment:
This might be an unnecessary breaking change if the method is called with parameter names before arguments, such as `GenerateImageAsync(description: "generate an image of....", ...)`. Do we truly need to rename it?